vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -496,8 +496,9 @@ def grouped_topk(hidden_states: torch.Tensor,
     else:
         raise ValueError(f"Unsupported scoring function: {scoring_func}")
 
+    original_scores = scores
```

Comment:
Could you leave a comment either here or at the topk/gather section why we need to save the original scores and separately gather from them